Assets/MRTK/Core/Providers/InputSimulation/MixedRealityInputSimulationProfile.cs

```diff
@@ -348,7 +348,29 @@ public class MixedRealityInputSimulationProfile : BaseMixedRealityProfile
         /// Apply random offset to the controller position
         /// </summary>
         public float ControllerJitterAmount => controllerJitterAmount;
+
+        [Header("Motion Controller Settings")]
         [SerializeField]
+        [Tooltip("Key to simulate a trigger press (select) on the motion controller")]
```

Comment:
Nit: I know this section is mimicing previous ones - there's not a lot of space between each property. It could be worth adding some whitespace between each couple of things (i.e. 

```
[SerializeField]
        [Tooltip("Key to simulate a menu press on the motion controller")]
        private KeyBinding motionControllerMenuKey = KeyBinding.FromKey(KeyCode.M);
        /// <summary>
        /// Key to simulate a menu press on the motion controller
        /// </summary>
        public KeyBinding MotionControllerMenuKey => motionControllerMenuKey;
```

to

```
[SerializeField]
        [Tooltip("Key to simulate a menu press on the motion controller")]
        private KeyBinding motionControllerMenuKey = KeyBinding.FromKey(KeyCode.M);

        /// <summary>
        /// Key to simulate a menu press on the motion controller
        /// </summary>
        public KeyBinding MotionControllerMenuKey => motionControllerMenuKey;
```